test/browser/support/TestHarness.js

```diff
@@ -84,89 +84,63 @@ module.exports = class TestHarness {
       keyList = [];
       for (i = 0; i < keys.length; i++) {
         keyList.push(
-          listObjectsParsedResponse?.ListBucketResult?.Contents.at(i).Key
+          listObjectsParsedResponse?.ListBucketResult?.Contents.at(i).Key,
         );
       }
     } while (keys.length < txmaEventSize);
     return keyList;
   }
 
-  async validateTxMAEventData(keyList, journeyType) {
+  async getTxMAEventData(keyList) {
+    let obj = {};
     let i;
-    let valid = Boolean;
-    
     for (i = 0; i < keyList.length; i++) {
-      const getObjectResponse = await this.HARNESS_API_INSTANCE.get(
+      const txmaEventBody = await this.HARNESS_API_INSTANCE.get(
         "/object/" + keyList[i],
-        {}
+        {},
       );
-      console.log(JSON.stringify(getObjectResponse.data, null, 2));
-      const eventName = getObjectResponse.data.event_name;
-      const Ajv = require("ajv").default;
-      const AjvFormats = require("ajv-formats");
-      const ajv = new Ajv({ strictTuples: false });
+      console.log(JSON.stringify(txmaEventBody.data, null, 2));
+      const eventName = txmaEventBody.data.event_name;
+      obj[eventName] = txmaEventBody.data;
+    }
+    return obj;
+  }
 
-      AjvFormats(ajv);
+  async validateTxMAEventData(allTxmaEventBodies, eventName, schemaName) {
+    const Ajv = require("ajv").default;
+    const AjvFormats = require("ajv-formats");
+    const ajv = new Ajv({ strictTuples: false });
+    ajv.addSchema(
+      CIC_CRI_AUTH_CODE_ISSUED_SCHEMA,
+      "CIC_CRI_AUTH_CODE_ISSUED_SCHEMA",
+    );
+    ajv.addSchema(CIC_CRI_END_SCHEMA, "CIC_CRI_END_SCHEMA");
+    ajv.addSchema(
+      CIC_CRI_START_BANK_ACCOUNT_SCHEMA,
+      "CIC_CRI_START_BANK_ACCOUNT_SCHEMA",
+    );
+    ajv.addSchema(CIC_CRI_START_SCHEMA, "CIC_CRI_START_SCHEMA");
+    ajv.addSchema(CIC_CRI_VC_ISSUED_SCHEMA, "CIC_CRI_VC_ISSUED_SCHEMA");
```

Comment:
Do we want to be doing this every time the `validateTxMAEventData` function is run or is it something that can be done once at the top of the file?